storage/remote/client_test.go

```diff
@@ -73,7 +74,7 @@ func TestStoreHTTPErrorHandling(t *testing.T) {
 			t.Fatal(err)
 		}
 
-		err = c.Store(&prompb.WriteRequest{})
+		err = c.Store(context.TODO(), &prompb.WriteRequest{})
```

Comment:
I'd use Background for a test, it's not as if we plan on plumbing through a better context at some point.